### DISCUSSION TITLE
Audio Quality

### DISCUSSION THREAD

**Discussion by markus681** [mkb79/audible-cli#18] [Q&A]
Hey, is there any chance for higher Audio Quality? I saw something with Extreme in one of your files.
Or is LC_128_44_Stereo the only Codec they provide? If i convert the AAX file to MP3 there is 128k obviously...if i convert it to flac i get about 600-800k...is there a way to get higher quality on MP3?

Is there a way to get Information about Quality? Like Charles Proxy?

Thanks

Greetz Markus

**Comment by mkb79** [comment]
Hi,

Audible provide audiobooks in different quality. If you use download books with audible-cli you can select a quality with the `-q` option. Default option is `best`.  `best` is the same quality like `extreme`. 

If you download audiobooks in `best` quality, you are getting the best available. For most audiobooks best is 128k. For some other 64k is the best quality provided by Audible. If your audio player support `AAC` don't convert them to MP3, use ffmpeg to remove encryption only. You can select a higher mp3 Bitrate using ffmpeg (read the guide for ffmpeg). But a higher Bitrate is useless if the source has a lower Bitrate.
